native/src/handler.js

```diff
@@ -0,0 +1,21 @@
+import { guessParsing } from "./parser";
+import { error, ok } from "./response";
+
+function parse(data) {
+  try {
+    let { content } = JSON.parse(data)
+    let ast = guessParsing(content);
+    return  ok(ast);
+  } catch (ex) {
+    return error(ex);
+  }
+}
+
+function jsonify(data) {
+  return `${JSON.stringify(data)}
+`;
+}
+
+export function handler(input) {
```

Comment:
This has to do with rollup. Using multiple `export default`s is a bit of a pain in the ass if you want to get it. I was planning on moving test to be built via rollup as well, but several configs mean several files. Delaying for a bit.